source/operator.rst

```diff
@@ -1026,7 +984,7 @@ Percona XtraDB Cluster backups.
 +-----------------+-------------------------------------------------------------------------------------------+
 |                                                                                                             |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Key**         | backup.affinity.tolerations                                                               |
+| **Key**         | backup.storages.<storage-name>.affinity.tolerations                                       |
```

Comment:
tolerations is not a part of affinity

Reply:
you can check https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml